db/schema.rb

```diff
@@ -507,73 +510,73 @@
-    t.string   "unconfirmed_email",                  limit: 255
+    t.string   "unconfirmed_email"
     t.string   "confirm_email_token",                limit: 30
     t.datetime "locked_at"
-    t.boolean  "show_community_spotlight_in_stream",               default: true,  null: false
-    t.boolean  "auto_follow_back",                                 default: false
-    t.integer  "auto_follow_back_aspect_id",         limit: 4
-    t.text     "hidden_shareables",                  limit: 65535
+    t.boolean  "show_community_spotlight_in_stream",             default: true,  null: false
+    t.boolean  "auto_follow_back",                               default: false
+    t.integer  "auto_follow_back_aspect_id"
+    t.text     "hidden_shareables"
     t.datetime "reset_password_sent_at"
     t.datetime "last_seen"
     t.datetime "remove_after"
-    t.string   "export",                             limit: 255
+    t.string   "export"
     t.datetime "exported_at"
-    t.boolean  "exporting",                                        default: false
-    t.boolean  "strip_exif",                                       default: true
-    t.string   "exported_photos_file",               limit: 255
+    t.boolean  "exporting",                                      default: false
+    t.boolean  "strip_exif",                                     default: true
+    t.string   "exported_photos_file"
     t.datetime "exported_photos_at"
-    t.boolean  "exporting_photos",                                 default: false
-    t.string   "color_theme",                        limit: 255
+    t.boolean  "exporting_photos",                               default: false
+    t.string   "color_theme"
   end
 
   add_index "users", ["authentication_token"], name: "index_users_on_authentication_token", unique: true, using: :btree
-  add_index "users", ["email"], name: "index_users_on_email", length: {"email"=>191}, using: :btree
-  add_index "users", ["invitation_service", "invitation_identifier"], name: "index_users_on_invitation_service_and_invitation_identifier", unique: true, length: {"invitation_service"=>64, "invitation_identifier"=>nil}, using: :btree
+  add_index "users", ["email"], name: "index_users_on_email", using: :btree
+  add_index "users", ["invitation_service", "invitation_identifier"], name: "index_users_on_invitation_service_and_invitation_identifier", unique: true, using: :btree
   add_index "users", ["invitation_token"], name: "index_users_on_invitation_token", using: :btree
-  add_index "users", ["username"], name: "index_users_on_username", unique: true, length: {"username"=>191}, using: :btree
+  add_index "users", ["username"], name: "index_users_on_username", unique: true, using: :btree
 
   add_foreign_key "aspect_memberships", "aspects", name: "aspect_memberships_aspect_id_fk", on_delete: :cascade
   add_foreign_key "aspect_memberships", "contacts", name: "aspect_memberships_contact_id_fk", on_delete: :cascade
```

Comment:
Since it's the last commit, we can fairly easily edit it:

```
git checkout upstream/develop -- db/schema.rb
git commit --amend db/schema.rb
git push -f origin 5813-show-geolocation-on-osm
```

This will replace the last commit, so the other one of you will have to `git fetch` and `git reset --hard origin/5813-show-geolocation-on-osm` their branch, like after a rebase by somebody else.